conf/check_leader_stability_in_compaction.json

```diff
@@ -0,0 +1,238 @@
+        },
+        {
+            "type": "CreateSpaceAction",
+            "space_name": "check_leader_stability_in_compaction",
+            "replica": 3,
+            "parts": 100,
+            "depends": [6]
+        },
+        {
+            "type": "UseSpaceAction",
+            "space_name": "check_leader_stability_in_compaction",
+            "depends": [7]
+        },
+        {
+            "type": "CreateSchemaAction",
+            "name": "t",
+            "props": [
+                {"name": "name", "type": "string"}
+            ],
+            "edge_or_tag": false,
+            "depends": [8]
+        },
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 10000,
+            "depends": [9]
+        },
+        {
+            "type": "SetFlagAction",
+            "layer": "storage",
+            "name": "disable_auto_compactions",
+            "value": "true",
+            "depends": [10]
+        },
+        {
+            "type": "SetFlagAction",
+            "layer": "storage",
+            "name": "wal_ttl",
+            "value": "60",
+            "depends": [11]
+        },
+        {
+            "type": "BalanceLeaderAction",
+            "depends": [12]
+        },
+        {
+            "type": "CheckLeadersAction",
+            "expected_num": 100,
+            "space": "check_leader_stability_in_compaction",
+            "depends": [13]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "t",
+            "col": "name",
+            "total_rows": 3000000,
+            "row_size": 1000,
+            "start_id": 1,
+            "depends": [14]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "t",
+            "col": "name",
+            "total_rows": 3000000,
+            "row_size": 1000,
+            "start_id": 3000001,
+            "depends": [14]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "t",
+            "col": "name",
+            "total_rows": 3000000,
+            "row_size": 1000,
+            "start_id": 6000001,
+            "depends": [14]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "t",
+            "col": "name",
+            "total_rows": 3000000,
+            "row_size": 1000,
+            "start_id": 9000001,
+            "depends": [14]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "t",
+            "col": "name",
+            "total_rows": 3000000,
+            "row_size": 1000,
+            "start_id": 12000001,
+            "depends": [14]
+        },
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 120000,
+            "depends": [15, 16, 17, 18, 19]
+        },
+        {
+            "type": "CheckLeadersAction",
+            "expected_num": 100,
+            "space": "check_leader_stability_in_compaction",
+            "result_var_name": "leaderDis1",
```

Comment:
$leaderDis1 ?

Reply:
Here is the variable name, not the variable， so is leaderDis1 better?